frontend/src/components/Login.vue

```diff
@@ -5,10 +5,10 @@
         <v-flex xs12 sm6>
           <v-alert
             type="error"
-            :value="error_msg ? true : false"
-            @click="error_msg = ''"
+            :value="getError ? true : false"
```

Comment:
I assume you want to convert `getError` variable to a boolean value, so maybe you could use:
```suggestion
            :value="!!getError"
```